test/test_rcm2points.py

```diff
@@ -168,3 +168,18 @@ def test_rcm2points_float32_msg_opt2(self):
                        lon,
                        opt=2,
                        msg=msg32))
+
+
+class Test_rcm2points_xr(ut.TestCase):
+    """Test_rcm2points This unit test covers the nominal, nan, and msg cases of
+    64 and 32 bit float input for rcm2point for xarray.DataArray inputs."""
+
+    def test_rcm2points_float64_nom_opt0(self):
+        nt.assert_array_almost_equal(
+            xr.DataArray(fo_nom_opt0_expected),
+            rcm2points(lat2d,
```

Comment:
Would prefer if this was returned to a variable before being compared with np.assert... willing to merge as is.

Reply:
Done for all the cases in this file.

Reply:
Thanks, resolved.